plugins/@grouparoo/spec-helper/playwright.config.ts

```diff
@@ -0,0 +1,111 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from "@playwright/test";
+
+const port = 30000 + parseInt(process.env.TEST_WORKER_INDEX ?? "0");
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  globalSetup: "./playwrightGlobalSetup",
```

Comment:
@evantahler did not combine the globalSetup as those have a little more nuance to them than the playwright config.